pages.es/common/aapt.md

```diff
@@ -0,0 +1,17 @@
+# aapt
+
+> Herramienta para empaquetado de activos de Android.
+> Compila y empaqueta recursos de una app de Android.
+> Más información: <https://elinux.org/Android_aapt>.
+
+- Lista los archivos contenidos en un archivo APK:
+
+`aapt list {{ruta/a/la/app.apk}}`
```

Comment:
So is this `path/to/the/app.apk`?

Reply:
That's right

Reply:
Do you want Spanish classes ? lol `the` could be `el` or `la` in Spanish based on the gender. and `to the` is `al` (contraction for: `a el`) or `a la`.

Reply:
I don't think that's correct though.  We don't use `path/to/the/...` in any other pages 🤔

Reply:
@bl-ue you're right
Please @Nacho-source use 
```suggestion
`aapt list {{ruta/a/app.apk}}`
```

Reply:
@navarroaxel I appreciate knowing that—thank you.  I plan to learn Spanish eventually, so some time I'll stop by at Buenos Aires and we can have a long conversation together! :D

So there's no formal way to indicate contractions in Spanish?  In English, you know, we have an apostrophe (`'`) for that, e.g. `don't` → `do not`, `can't` → `cannot` etc.

Reply:
No no.
He means that the formal ways to make contractions in Spanish are `del` (-> de el ) or `al` (--> a el), respectively.
That is part of the language.
But if you want to, @navarroaxel could give you some informal Spanish lessons too. :laughing: